src/main/java/htsjdk/variant/variantcontext/VariantContextBuilder.java

```diff
@@ -28,16 +28,7 @@
 import htsjdk.variant.vcf.VCFConstants;
 
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.EnumSet;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Using wildcard imports makes it harder to figure out where a name came from when reading the code outside of an IDE. Changing an IDE setting can enforce the rule to keep imports, instead of replacing them with a wildcard.